package.json

```diff
@@ -599,34 +599,35 @@
 		"test": "npm run compile && node ./out/test/runTest.js"
 	},
 	"devDependencies": {
-		"@types/mocha": "^5.2.7",
-		"@types/node": "8.10.18",
-		"@types/vscode": "^1.44.0",
 		"@types/fs-extra": "^8.1.0",
+		"@types/mocha": "^5.2.7",
+		"@types/node": "^14.14.6",
+		"@types/vscode": "^1.50.0",
+		"chai": "^4.2.0",
 		"del": "^4.1.1",
 		"event-stream": "^4.0.1",
 		"gulp": "^4.0.2",
+		"gulp-cli": "^2.3.0",
 		"gulp-filter": "^5.1.0",
 		"gulp-sourcemaps": "^2.6.5",
 		"gulp-typescript": "^5.0.1",
-		"run-sequence": "^2.2.1",
-		"tslint": "5.11.0",
-		"typescript": "3.2.4",
-		"vscode-test": "^1.3.0",
 		"mocha": "^6.1.4",
-		"chai": "^4.2.0",
+		"run-sequence": "^2.2.1",
 		"source-map-resolve": "0.6.0",
-		"vscode-nls-dev": "^3.2.6"
+		"tslint": "5.11.0",
+		"typescript": "^3.9.7",
```

Comment:
So this will resolve the issue https://github.com/Autodesk-AutoCAD/AutoLispExt/issues/31